scenes/song_parser/song.gd

```diff
@@ -1,29 +1,18 @@
-extends Node
 class_name Song
 
 
-@export var artistName = ""
-@export var title = ""
-@export var artistNameSort = ""
-@export var artistYear = ""
-@export var artistXXXXYear = ""
```

Comment:
I was getting compiler warnings before about, if stuff isnt exported you cant use this Song class effectively in other classes

Reply:
Do you know in what context that was? Export tooltips are only useful when you want to edit the values of a node or Resource in the inspector but we're doing neither of those things.

Reply:
Additionally, @export doesn't work when a script isn't a Node or Resource, it's neither in this PR.